### PR TITLE
feat: add new icons

### LINKED ISSUE #109: feat: Add new icons
**Here are new icons to add (10):**

- [ ] [volume-alert-line & volume-alert-fill](https://github.com/Decathlon/vitamin-design/issues/152)
image

- [ ] [shopping-bag-line & shopping-bag-fill](https://github.com/Decathlon/vitamin-design/issues/145)
image

- [ ] [seed-1-line & seed-1-fill](https://github.com/Decathlon/vitamin-design/issues/139)
image

- [ ] [seed-2-line & seed-2-fill](https://github.com/Decathlon/vitamin-design/issues/140)
image

- [ ] [seed-3-line & seed-3-fill](https://github.com/Decathlon/vitamin-design/issues/141)
image

### PR DESCRIPTION
**Pull request by daniel-dumortier**
## Changes description
Add new icons in fill and line variant: 
- seed-1
- seed-2
- seed-3
- shopping-cart
- volme-alert

## Context
closes #109

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on an iPhone device/simulator.
- [x] I have tested on an iPad device/simulator.
- [x] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

## Does this introduce a breaking change?

- No

#### iPhone

Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 38 33
Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 37 17
Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 38 27
Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 38 11
Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 37 44
Simulator Screen Shot - iPhone 8 - 2023-02-06 at 16 36 59

#### iPad

icons are not dependent on platform

### CODE REVIEW

**Review by lauthieb — DISMISSED** [review]

**Comment by lauthieb** [comment]
Thank you very much @daniel-dumortier !

**Review by baptistedajon — APPROVED** [review]

File: Sources/VitaminCore/Foundations/Icons/Vitamix.swift [hunk 1]
```diff
@@ -409,8 +415,10 @@ public enum Vitamix {
       public static let play = VitaminImageAsset(name: "line/media/play")
       public static let playCircle = VitaminImageAsset(name: "line/media/play_circle")
       public static let `repeat` = VitaminImageAsset(name: "line/media/repeat")
+      public static let shoppingBag = VitaminImageAsset(name: "line/media/shopping_bag")
```
Comment (florentlotthepro) [inline Sources/VitaminCore/Foundations/Icons/Vitamix.swift:418]: Do we need this one? It's already in finance

**Review by florentlotthepro — DISMISSED** [review]
Please check the shopping bag in Line Media

File: Sources/VitaminCore/Foundations/Icons/Vitamix.swift (hunk 1 above)
Comment (daniel-dumortier) [reply Sources/VitaminCore/Foundations/Icons/Vitamix.swift]: you're right, I think I made a wrong drag and drop, thought it had not work, and reimported the icon in right section, before running Swiftgen.
Fixed !

**Review by florentlotthepro — APPROVED** [review]
👌

**Pull request merged**